ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -240,6 +240,16 @@ Item {
         //            }
         //        }
 
+        Connections {
+            target: chatLogView.model
+            onDataChanged: {
+                let item = chatLogView.itemAtIndex(topLeft.row)
```

Comment:
@borismelnik we should prevent triggering this slot in case of emitting `dataChanged` signal from `message_model.nim` for all cases but the case in the line `421`. Try to check if role `ResponseToMessageWithId` has been changed or not.